lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -153,8 +155,10 @@ sub create {
     my $self = $class->SUPER::create(@_);
     return unless ($self);
 
-    my $rv = eval {
+    $self->_lock_per_lane_alignment();
+    my @temp_allocations = ();
 
+    try {
         #TODO In a future version collect relevant alignments from other merged alignment results when available
```

Comment:
I don't think so, but I don't know the original intent of the TODO! I believe it is quite old.